department-of-veterans-affairs/va.gov-cms
# VAGOV-5409: Metatags for outreach product.

QA: 


Check sitename default metatag
* Go to /admin/config/system/site-information
* note the site name is now Veterans Affairs
* go to admin/config/search/metatag, and note that the default (sitewide and content) metatag uses [site:name]

New meta title config
* Create an office, publication_listing, and event_listing node. 
* Fill out the meta title tag field for each content type. 
* Hit save. The `<title>` meta should use the meta title tag field. 

Cross check content types that have the additional "field_meta_title"
* go to /admin/reports/fields
* note that field_meta_title is concurrently available on the following content types: Benefits detail pageBenefits hub landing pageHealth Care SystemDetail PageHealth Care FacilityPublication listingEvent listingOffice. 
* go to admin/config/search/metatag and note that the field_meta_title is being used to generate the meta title for each of the above content types (the content type labels are not always totally equivalent, for some reason)
